common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -136,16 +150,29 @@ export class SendTransaction extends React.Component {
                   <AmountField
                     value={value}
                     unit={unit}
+                    tokens={this.props.tokenBalances
+                      .filter(token => !token.balance.eq(0))
+                      .map(token => token.symbol)
+                      .sort()}
```

Comment:
This is actually from the token branch PR -- I actually commented the same thing hehe. https://github.com/MyEtherWallet/MyEtherWallet/pull/47#discussion_r127576383